### ISSUE TITLE
Mis-configured Docker tag/versioning

### ISSUE THREAD

**Issue by staskh** [miniwdl-ext/miniwdl-aws#11]
Docker build script in .github/workflows/ghcr_image.yml uses GIT versioning  as a docker tag which, unless commit has an explicit  tag, different from one  generated by  version.py  and stored in RELEASE-VERSION. For example, while RELEASE-VERSION has version 0.4.0.post24.dev208077062, docker image will be  generated  with tag v0.4.0-24-gc670106.

Suggested fix: replace line 25 in .github/workflows/ghcr_image.yml  with  
```
TAG="v$(cat RELEASE-VERSION)"
```

**Comment by mlin** [comment]
@staskh you're right that this is an undesirable inconsistency between RELEASE-VERSION and docker tags (*when there are additional commits since the last tagged version). I personally find the git-generated ones more useful though, in that you can read the git commit hash it's built from, whereas that information is [obscured](https://github.com/miniwdl-ext/miniwdl-aws/blob/2fd4d6d4b9a51fbc9abc8adb6493d76e211ef844/version.py#L86) in the RELEASE-VERSION tag. So my first inclination would actually be to go the other direction and change RELEASE-VERSION. This might cause problems with PyPI, but my practice is only to push tagged release versions to PyPI anyway. Would this cause any issues for you?
